news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Call for paper for the PrestaShop Day Paris 2020"
```

Comment:
```suggestion
title:  "Call for Papers curently open for the PrestaShop Day Paris 2020"
```

Reply:
Nice catch, I forgot the "s" to "Papers".
What do you think of:
```suggestion
title:  "Call for Papers for the PrestaShop Day Paris 2020 is open"
```

Reply:
"for" "for", it's redundant

Reply:
CFP for the PrestaShop Day Paris 2020 is open
?

Reply:
CFP isn't explicit for everyone.

Reply:
```suggestion
title:  "PrestaShop Day Paris 2020: Call for paper is open"
```

Reply:
```suggestion
title:  "PrestaShop Day Paris 2020: Call for Papers is open"
```